apps/bot/src/services/twitch-api/api-connector.ts

```diff
@@ -34,7 +34,7 @@ export async function getChannelRefreshKey(code: string): Promise<TOption<TTwitc
     }`;
 
     const res = await fetch(url, { method: "POST" });
-    const json = await res.json<TTwitchOAuthRefresh>();
+    const json = (await res.json()) as TTwitchOAuthRefresh;
```

Comment:
I'll add on this one only, but it's refers to all changes in general. 
Making the usage of the `as Type` is a little bit dangerous. Since you have the type, maybe you could try to use `zod` to be sure that you have the right stuff at your hand?